### PR TITLE
16 contributing guidelines

### PR DESCRIPTION
**Pull request by tomseli**
Added contributing guidelines for the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,60 @@
...
+Header files: `\inc\*.h`
+
+The name of the file must be in snake_case and cover its contents. It's encouraged to keep files as simple as possible. 
+
+### Naming convention
+All *functions* part of the API must contain the prefix `API_`. 
+
+Other important prefixes:
+- `LL_`, Logic Layer
+- `AL`, Aplication Layer
```
Comment (Mirkoet) [inline README.md:62]: `AL_ ` Spelling error

**Review by Mirkoet — CHANGES_REQUESTED** [review]
See comments

File: README.md [hunk 2]
```diff
@@ -3,4 +3,60 @@
...
+    - Crosscheck code with the issue description!
+
+After all these checks are passed, the issue owner should merge the branch with the main branch.
+
+## Other standards
+
+### Files
+All files must follow the following naming convention:
+
+Source files: `\src\*.cpp\` or `\src\*.c`
```
[the remark this answers is not in the document]
Comment (tomseli) [reply README.md]: I agree. Adjusted.

File: README.md [hunk 3]
```diff
@@ -3,4 +3,60 @@
...
+An example: `#16 fixed stuck in inf loop in API_Draw_line`.
+
+The issue number links the commit to the relevant issue and adds it to its history.
+
+## Merging
+After the issue is deemed complete, it must be merged via a pull request (PR). A PR must pass several  checks (some automatic) before it can be merged into the main branch:
+
+1. (Automatic) The project must build
+2. (Automatic) The unit tests must pass (WIP)
+3. (Automatic) The style checker must pass (WIP)
```
[the remark this answers is not in the document]
Comment (tomseli) [reply README.md]: Removed WIP as #24 is ready to be merged anyway

File: README.md [hunk 4]
```diff
@@ -3,4 +3,63 @@
...
+Header files: `*.h`
+
+The name of the file must be in snake_case and cover its contents. It's encouraged to keep files as simple as possible. 
+
+### Naming convention
+All *functions* part of the API must contain the prefix `API_`. 
+
+Other important prefixes:
+- `BL_`, Business Layer
+- `AL`, Application Layer
```
Comment (Mirkoet) [inline README.md:65]: `AL_`

**Review by Mirkoet — CHANGES_REQUESTED** [review]
MINOR SPELLING MISTAKE, REQUEST CHANGES

File: README.md [hunk 5]
```diff
@@ -3,4 +3,63 @@
...
+## Other standards
+
+### Files
+All files must follow the following naming convention:
+
+Source files: `*.cpp\` or `*.c`
+
+Header files: `*.h`
+
+The name of the file must be in snake_case and cover its contents. It's encouraged to keep files as simple as possible.
```
Comment (Mirkoet) [inline README.md:58]: What does this mean? Write simple code? or keep as little as possible in the file?

**Review by Mirkoet — CHANGES_REQUESTED** [review]
See comment

**Review by Mirkoet — APPROVED** [review]

**Pull request merged**